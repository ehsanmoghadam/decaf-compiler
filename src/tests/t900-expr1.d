void main () {
    int a;
    a = 12;
    a += 10;
    Print(a);
    a -= 10;
    Print(a);
    a *= 10;
    Print(a);
    a /= 10;
    Print(a);
}