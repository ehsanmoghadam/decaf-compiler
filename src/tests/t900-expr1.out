22
12
120
12